src/Library/demos/Row/main.blp

```diff
@@ -0,0 +1,104 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Rows";
+  description: "Create user interface lists with ease";
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+      Adw.PreferencesGroup  {
```

Comment:
can you fix the indentaton of the file?

Reply:
> can you fix the indentaton of the file?

Sure, it would be really nice if Workbench support autoindentation one day